pkgs/by-name/ru/ruffle/package.nix

```
@@ -1,102 +1,74 @@
 {
-  alsa-lib,
+  lib,
+  stdenvNoCC,
+  rustPlatform,
+  withRuffleTools ? false,
   fetchFromGitHub,
-  makeWrapper,
-  openssl,
+  jre_minimal,
   pkg-config,
-  python3,
-  rustPlatform,
-  stdenvNoCC,
-  lib,
+  wrapGAppsHook3,
+  darwin,
+  alsa-lib,
+  gtk3,
+  openssl,
   wayland,
-  xorg,
   vulkan-loader,
   udev,
-  jre_minimal,
-  cairo,
-  gtk3,
-  wrapGAppsHook3,
-  gsettings-desktop-schemas,
-  glib,
   libxkbcommon,
   openh264,
-  darwin,
+  versionCheckHook,
+  nix-update-script,
 }:
-let
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "ruffle";
-  version = "nightly-2025-01-25";
-  # TODO: Remove overridden derivation once ruffle accepts upstream openh264-2.5.0
-  openh264-241 =
-    if stdenvNoCC.hostPlatform.isLinux then
-      openh264.overrideAttrs (_: rec {
-        version = "2.4.1";
-        src = fetchFromGitHub {
-          owner = "cisco";
-          repo = "openh264";
-          tag = "v${version}";
-          hash = "sha256-ai7lcGcQQqpsLGSwHkSs7YAoEfGCIbxdClO6JpGA+MI=";
-        };
-        postPatch = null;
-      })
-    else
-      null;
-in
-rustPlatform.buildRustPackage {
-  inherit pname version;
+  version = "nightly-2025-03-16";
```

```suggestion
  version = "0-nightly-2025-03-16";
```
There is a new policy that all versions must start with a digit.